pkgs/development/go-modules/generic/default.nix

```diff
@@ -126,7 +126,7 @@ let
     inherit (go) GOOS GOARCH;
 
     GO111MODULE = "on";
-    GOFLAGS = "-mod=vendor";
+    GOFLAGS = "-mod=vendor -trimpath";
```

Comment:
Does this make `find $out/{bin,libexec,lib} -type f 2>/dev/null | xargs -r ${removeExpr removeReferences} || true` obsolete or what paths does it remove?

Reply:
Yes, we should be able to drop `removeReferences`.

I started working on this on a different branch and thought I'd combine them to save a few rebuilds but I seem to have messed them up. I'll need to fix it and retest.